unstructured/nlp/patterns.py

```diff
@@ -33,3 +33,14 @@
     "·",
 ]
 UNICODE_BULLETS_RE = re.compile(f"({'|'.join(UNICODE_BULLETS)})")
+
+IP_ADDRESS_PATTERN = "[0-9]{1,2}\.[0-9]{1,2}\.[0-9]{1,2}\.[0-9]{1,2}","[a-z0-9]{4}::[a-z0-9]{4}:[a-z0-9]{4}:[a-z0-9]{4}:[a-z0-9]{4}%?[0-9]*"
+IP_ADDRESS_PATTERN_RE= re.compile(f"({'|'.join(IP_ADDRESS_PATTERN)})")
+
```

Comment:
Not sure if we want explicit patterns for ip address and ip name and date or if I should use the `extract_text_after` and `extract_text_before` methods. It seems like the `Received` follows the same pattern.

Reply:
I think it's good to have explicit patterns for each of these in there. You can use `extract_text_before` and `extract_text_after` too, but these patterns could come in handy outside of email parsing.